### ISSUE TITLE
Implement the claimable balance support in our demo client/wallet

### ISSUE THREAD

**Issue by Zagan202** [stellar/stellar-demo-wallet#34]
### What problem does your feature solve?

In order to leverage the new Protocol 14/15 feature claimable balances we need wallets to query for accounts that have claimable balances to claim.

### What would you like to see?

Notify Wallet users/accounts when a claimable balance is available for them to claim.

**Comment by Zagan202** [comment]
We should also deprecate the original demo client and only add new features to the new one.
This may need to be separated into another issue(s) entirely.

**Comment by tyvdh** [comment]
What is "the original demo client"?

Also another aside is does it make sense to keep this repo for anything other than the tutorial now that the new account viewer is nearly ready? Should we be using that for all future demo/reference stuff? This may be a bigger conversation but I think it's an important one to have before we dupe too much between this repo and the new AV.

cc: @morleyzhi

**Comment by Zagan202** [comment]
Hey @tyvdh, I need to touch base with @JakeUrban when he gets back to specify which client "demo client" he's [referring to](https://github.com/stellar/stellar-protocol/issues/747#issuecomment-712447987) when he comes back on monday.

I believe we're referring to https://github.com/stellar/sep24-demo-client 

> Should we be using that for all future demo/reference stuff? 

The goal we would like to achieve is use the stellar-demo-wallet and polaris as an end to end experience demo and reduce duplication. If we want to extend the or edit either SEP we dont want to change both stellar-demo-wallet and sep24-demo-client.

**Comment by Zagan202** [comment]
> This may be a bigger conversation but I think it's an important one to have before we dupe too much between this repo and the new AV.

Im open to have this discussion with all relevant contributors sometime next week.
I agree we should be all on the same page for any big changes.

**Comment by JakeUrban** [comment]
Hi @tyvdh, can you explain your reasoning here? Account Viewer is not really a mock wallet that supports our SEP protocols, its an exploratory tool for looking at accounts on Stellar, so in my team's view they're completely disjoint areas.

@Zagan202 you're right, https://github.com/stellar/sep24-demo-client is the currently-used demo wallet but we need to transition to using this project for that purpose. 

Most of the ecosystem knows of the old demo wallet, so we need to ensure it's functional until they've transitioned to the new one. That transition process still needs to be worked out. I'm a fan of simply replacing the old client with the new one on the same domain once we're confident the new one is ready, whatever that means.

**Comment by tyvdh** [comment]
@brunomuler Is it true that the AV won't be implementing SEP protocols and that for demo/reference implementations this demo wallet should continue to be used, supported and improved alongside the new AV?
